peepingtom/peeper/peeper/tools/peep_together.py

```diff
@@ -0,0 +1,27 @@
+from ....io_ import read
+from ....core import DataCrate, ParticleBlock, stack
+from ...depictors import ParticleDepictor
+from ..base import Peeper
+
+
+def peep_particles_together(*file_paths):
+    """naively read multiple files containing particle and attempt to visualise common datacrates in the same viewer
+    """
+    file_contents = [read(file_path) for file_path in file_paths]
```

Comment:
`read` accepts an arbitrary number of file paths, so you can just do:
```
read(file_paths)
```
and you avoid having to do `file_contents[0]` later.